airbrakes/data_handling/logged_data_packet.py

```diff
@@ -21,137 +21,137 @@ class LoggedDataPacket(msgspec.Struct):
     invalid_fields: list[str] | None = None
 
     # Raw Data Packet Fields
-    scaledAccelX: float | None = None
-    scaledAccelY: float | None = None
-    scaledAccelZ: float | None = None
-    scaledGyroX: float | None = None
-    scaledGyroY: float | None = None
-    scaledGyroZ: float | None = None
-    deltaVelX: float | None = None
-    deltaVelY: float | None = None
-    deltaVelZ: float | None = None
-    deltaThetaX: float | None = None
-    deltaThetaY: float | None = None
-    deltaThetaZ: float | None = None
+    scaledAccelX: str | None = None
```

Comment:
why are we setting everything as a str?

Reply:
because we have to round the values. It was always doing that anyway. Now the type hint is correct.